bin/workflows/pycbc_offline

```diff
@@ -0,0 +1,355 @@
+
+final_veto_file, final_veto_name, ind_cats = \
+                                   wf.get_cumulative_veto_group_files(workflow,
+                                            'segments-final-veto-group',
+                                            veto_cat_files,
+                                            rdir['analysis_time/segment_data'],
+                                            execute_now=False)
+
+# setup gating files if provided
+gate_files = wf.setup_gating_workflow(workflow, output_dir="gating")
+
+# add the gate files to the jobs that use them
+for job in ['calculate_psd', 'inspiral', 'single_template',
+            'plot_singles_timefreq']:
+  for gate_file in gate_files:
+      ifo_gate = gate_file.cache_entry.url
+      try:
+          workflow.cp.set('{}-{}'.format(job, gate_file.ifo.lower()),
+                          'gating-file', ifo_gate)
+      except ConfigParser.SectionError:
+          workflow.cp.add_section('{}-{}'.format(job,
+                                  gate_file.ifo.lower()))
+          workflow.cp.set('{}-{}'.format(job, gate_file.ifo.lower()),
+                          'gating-file', ifo_gate)
+
+# Precalculated PSDs
+precalc_psd_files = wf.setup_psd_workflow(workflow, analyzable_segs,
+                                            datafind_files, "psdfiles")
+
+# Template bank stuff
+hdfbank = wf.setup_tmpltbank_workflow(workflow, analyzable_segs,
+                                      datafind_files, output_dir="bank",
+                                      psd_files=precalc_psd_files,
+                                      return_format='hdf')
+
+splitbank_files_fd = wf.setup_splittable_workflow(workflow, hdfbank,
+                                                  out_dir="bank",
+                                                  tags=['full_data'])
+splitbank_files_inj = wf.setup_splittable_workflow(workflow, hdfbank,
+                                                   out_dir="bank",
+                                                   tags=['injections'])
+
+bank_plot = [(wf.make_template_plot(workflow, hdfbank[0],
+              rdir['coincident_triggers']),)]
+
+# setup the injection files
+inj_files, inj_tags = wf.setup_injection_workflow(workflow,
+                                                  output_dir="inj_files")
+
+######################## Setup the FULL DATA run ##############################
+tag = output_dir = "full_data"
+ctags = [tag, 'full']
+
+# setup the matchedfilter jobs
+ind_insps = insps = wf.setup_matchedfltr_workflow(workflow, analyzable_segs,
+                                   datafind_files, splitbank_files_fd,
+                                   output_dir, tags = [tag])
+
+insps = wf.merge_single_detector_hdf_files(workflow, hdfbank[0],
+                                           insps, output_dir, tags=[tag])
+
+# setup sngl trigger distribution fitting jobs
+# 'statfiles' is list of files used in calculating coinc statistic
+statfiles = []
```

Comment:
I have a suspicion this is another function that can be removed, mostly writing this to remind me to look at this next week.